### PR TITLE
feat(packer): add packer configuration

### PR DESCRIPTION
**Pull request by xcixor**
# Description

This PR creates a packer base image for the app

Fixes # (issue)

## Screenshots
Screenshot from 2019-08-14 20-27-03

# Checklist

- [x] My Code **follows** the **style** guidelines **of** this **project**.
- [x] I have performed a **self**-review of **my** own code.
- [x] I have commented my code, particularly **in** hard-*to*-understand areas.
- [x] I have made **corresponding** changes **to** the documentation.
- [x] My changes generate **no new warnings**.
- [x] I have added tests that prove my fix **is** effective or that my feature works.
- [x] **New and** existing unit tests pass locally **with** my changes.
- [x] **Any** dependent changes have been merged **and** published **in** downstream modules.
- [x] **I have written commits as required, and did not merge develop to my branch. I rebased it instead**.

### CODE REVIEW

File: devops/pelly-base-image/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Building the Pelly Base Image
+## Prerequisites
+
+1. Ensure the following;
+- You have a [google cloud account](https://console.cloud.google.com/).
+- You have a [google cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+- You have enable the Compute Engine API.
```
Comment (JamesKirkAndSpock) [inline devops/pelly-base-image/README.md]: change to `enabled`

File: devops/pelly-base-image/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+2. Clone the repository as follows;
+
+ ```
+ git clone https://github.com/pelly-realtors/pelly-realtors-website.git
+ cd devops/pelly-base-image
+ ```
+
+3. Setup the service account;
+  Instructions to create a service account can be found [here](https://cloud.google.com/iam/docs/creating-managing-service-accounts). The role that the account will be able to perform is the most crucial is a crucial part of setting up the account. The minimum capabilities that the account should have are Compute Image User, Compute Instance Admin and Service Account Actor. Download the Service Account JSON key to your machine after giving the Service Account its roles.
```
Comment (JamesKirkAndSpock) [inline devops/pelly-base-image/README.md]: Fix the language logic on this line `able to perform is the most crucial is a crucial part of setting up the account.`

File: devops/pelly-base-image/packer.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "variables": {
+        "project_id": "{{env `PROJECT_ID`}}"
+      },
+
+    "builders": [
+      {
+        "type": "googlecompute",
+        "account_file": "../account/account.json",
+        "project_id": "{{user `project_id`}}",
+        "source_image": "ubuntu-1604-xenial-v20180306",
```
Comment (JamesKirkAndSpock) [inline devops/pelly-base-image/packer.json]: The source image is fine, try to research if there is an ubuntu 1604  xenial image for 2019

Comment (xcixor) [reply devops/pelly-base-image/packer.json]: This **ubuntu-1604-lts** is the most stable

Comment (xcixor) [reply devops/pelly-base-image/packer.json]: found this to be working **ubuntu-1604-xenial-v20190807**

File: devops/pelly-base-image/setup.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+install_dependencies () {
+    sudo apt-get -y upgrade
+    sudo apt-get -y update
+    sudo apt-get install -y python3-pip
```
Comment (pelly-realtors) [inline devops/pelly-base-image/setup.sh]: Install pip using wget not the apt-get one that installs the Precambrian pip package

**Pull request merged**